### PR TITLE
Upgrade to decidim 0 25

### PR DESCRIPTION
**Pull request by tramuntanal**
This PR upgrades this app to latest Decidim v0.25

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -14,10 +18,19 @@ class Application < Rails::Application
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
 
+    config.load_defaults 5.0
```
Comment (LauraJaimeC) [inline config/application.rb:21]: is this not 6.0?

File: decidim-census/app/views/layouts/census/application.html.erb [hunk 2]
```diff
@@ -2,8 +2,8 @@
 <html>
 <head>
   <title>Census</title>
-  <%= stylesheet_link_tag    "census/application", media: "all" %>
-  <%= javascript_include_tag "census/application" %>
+  <%= stylesheet_pack_tag "census/application", media: "all" %>
+  <%= javascript_pack_tag "census/application" %>
```
Comment (LauraJaimeC) [inline decidim-census/app/views/layouts/census/application.html.erb:6]: these files not exists in the module. In fact, packs/ not exists.

**Review by LauraJaimeC — CHANGES_REQUESTED** [review]
good job! only two details

File: decidim-census/app/views/layouts/census/application.html.erb (hunk 2 above)
Comment (tramuntanal) [reply decidim-census/app/views/layouts/census/application.html.erb:6]: This layout is not being used anywhere. Controllers in this module make use of Decidim's admin layout.
Do you think there's any need to change this layout?

Comment (LauraJaimeC) [reply decidim-census/app/views/layouts/census/application.html.erb:6]: Ok, let's just leave it then

**Comment by tramuntanal** [comment]
Superseded by https://github.com/diputacioBCN/decidim-diba/pull/118

**Pull request closed without merging**